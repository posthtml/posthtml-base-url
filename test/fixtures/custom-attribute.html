<div data-url="foo/bar.html"></div>
