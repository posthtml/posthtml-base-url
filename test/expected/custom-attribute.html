<div data-url="https://example.com/foo/bar.html"></div>
